modules/core/client/components/Map/map.less

```diff
@@ -0,0 +1,52 @@
+
+@import '~/node_modules/mapbox-gl/dist/mapbox-gl.css';
```

Comment:
nitpick: I don't think you need `node_modules` in the path here, it resolves inside, maybe just `~mapbox-gl/dist/mapbox-gl.css`.

Reply:
Fixed 👍

Reply:
Fixed 👍